### ISSUE TITLE
Allowing Pause on Search

### ISSUE THREAD

**Issue by eduardo4jesus** [htop-dev/htop#1494]
I currently use an update rate of  `1.5` seconds. However, when I search for a process to finalize and I need to navigate through the options (searching next or using the arrows up and down) the list updates while I am doing it, which moves things up and down in a confusing way, because of the new order of the processes.

Would it be possible to not update while in the search mode? Does that already exist?

If Not, would it be possible to add an option somewhere (maybe a short ut F-function) that I can signal to pause the update until I leave the search?

Many thanks.

**Comment by BenBE** [comment]
In normal operation `htop` already stops updating the sort order of processes when the list is navigated. This should also apply for the search AFAIK.

Which version of `htop` are you running (displayed top-left with <kbd>F1</kbd> while running or with `htop --version`)?

**Comment by eduardo4jesus** [comment]
I am using `htop 3.2.0`

**Comment by eduardo4jesus** [comment]
I don't mean sort by. I mean `F3 Search`, and also when I and navigating with arrows `Up` and `Down`.

**Comment by eduardo4jesus** [comment]
Just noticed that `Sorting by CPU` was enabled instead of `Sort By PID`, maybe that was causing the issue.

**Comment by BenBE** [comment]
No, even with sorting by CPU enabled the list will stop sorting for a little while when you navigate. This is independent of the search as it allows for better targeting processes. So if this is broken without searching too then there's something wrong. Can you check with a more recent version (3.3.0 or from `main`)?

**Comment by eduardo4jesus** [comment]
I just removed `htop` from my system and downloaded and installed the compiled from source. 

```
eduardoj@ThunderBit:~$ htop --version
htop 3.4.0-dev-3.3.0-193-g171f828
eduardoj@ThunderBit:~$ rm -rf ~/.config/htop/htoprc 
```

When I run the command `htop` and press the arrow down to navigate to a process in the middle of the list. After waiting a few seconds without pressing any key, the list is still updated.

What else could I try?

**Comment by eduardo4jesus** [comment]
![image](https://github.com/user-attachments/assets/a42892c3-083c-41d9-ae9b-3fec5dc035a9)

I am using `Z`, which does what I want. Not sure when this was implemented, but thank you so much.

**Comment by fasterit** [comment]
it was implemented for `htop` 3.0.3 and is fully functional from 3.0.4
